src/pymor/vectorarrays/numpy.py

```diff
@@ -506,6 +506,9 @@ def dofs(self, dof_indices):
     def amax(self):
         return self.base.amax(_ind=self.ind)
 
+    def __str__(self):
+        return self.base.__str__()
```

Comment:
This seems wrong, as the indices are neglected. Maybe better impelment `NumpyVectorArray.__str__` via `to_numpy`?

Reply:
Yes, of course. It should be fixed now. Maybe it's better to avoid `to_numpy` to avoid copying?

Reply:
I'm fine with the current implementation.